### PR TITLE
Form - Keep validation summary div in nested Forms

### PR DESCRIPTION
**Pull request by IgnatovDan**

### CODE REVIEW

**Review by andrewmakarov — DISMISSED** [review]

File: js/ui/scheduler/recurrence_editor.js [hunk 1]
```diff
@@ -431,7 +431,7 @@ class RecurrenceEditor extends Editor {
     _renderEditors($container) {
         this._recurrenceForm = this._createComponent($container, Form, {
             items: this._editors,
-            showValidationSummary: true,
+            showValidationSummary: false,
```
Comment (novsstation) [inline js/ui/scheduler/recurrence_editor.js:434]: Do we need render validation summary in Scheduler by default?

File: js/ui/form/ui.form.js [hunk 2]
```diff
@@ -314,18 +318,15 @@ const Form = Widget.inherit({
     },
 
     _renderValidationSummary: function() {
-        const $validationSummary = this.$element().find('.' + FORM_VALIDATION_SUMMARY);
-
-        if($validationSummary.length > 0) {
-            $validationSummary.remove();
-        }
+        this._$validationSummary?.remove();
+        this._$validationSummary = undefined;
```
Comment (novsstation) [inline js/ui/form/ui.form.js:321]: I think it is better to move 2 lines above to a new method disposeValidationSummary.
Something like this:

``` js
 function disposeValidationSummary() {
        this._$validationSummary?.remove();
        this._$validationSummary = undefined;
}

Or maybee it is unnecessary?

File: js/ui/scheduler/recurrence_editor.js (hunk 1 above)
Comment (IgnatovDan) [reply js/ui/scheduler/recurrence_editor.js:434]: In dxForm, there will be a div with fixed size that reserves empty space when validationSUmmary is enabled.
The root Appointment form have this option enabled.
But the nested Recurrence editor, that internally uses the dxForm, doesn't declare validation rules and should not reserve that space for summary.

**Review by novsstation — APPROVED** [review]

File: js/ui/form/ui.form.js (hunk 2 above)
Comment (IgnatovDan) [reply js/ui/form/ui.form.js:321]: You are right and I have fixed this code duplication.

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**